nifi-commons/nifi-web-client-api/src/main/java/org/apache/nifi/web/client/api/HttpResponseEntity.java

```diff
@@ -43,4 +44,11 @@ public interface HttpResponseEntity extends Closeable {
      * @return HTTP Response Body stream can be empty
      */
     InputStream body();
+
+    /**
+     * Get the endpoint URI that was accessed to generate this HTTP response
+     *
+     * @return HTTP URI from which the response was retrieved
+     */
+    URI getUri();
```

Comment:
I should have noticed this earlier, but following the convention of the other methods, this method should be named `uri()` instead of `getUri()`.
```suggestion
    URI uri();
```